### PR TITLE
Parse.Cloud.onLiveQueryEvent Documentation

### LINKED ISSUE #628: 📕 Parse.Cloud.onLiveQueryEvent not documented
__Link to section:__
https://github.com/parse-community/parse-server/blob/master/src/cloud-code/Parse.Cloud.js#L290

__What is the issue?__
Parse.Cloud.onLiveQueryEvent  not documented

__Can you propose a solution?__
https://stackoverflow.com/questions/43708317/detect-livequery-connected-clients-on-parse-server

### PR DESCRIPTION
**Pull request by dplewis**
closes: https://github.com/parse-community/docs/issues/628

This is a work in progress. I plan on adding more parameters on the server like sessionToken and connectedAt timestamp.

If you can think of any other parameters / events that would help with monitoring I can open a PR with them

### CODE REVIEW

File: _includes/cloudcode/cloud-code.md [hunk 1]
```diff
@@ -469,6 +469,37 @@ Parse.Cloud.beforeLogin(async request => {
 - On sign up
 - If the login credentials are incorrect
 
+# LiveQuery Triggers
+
+*Available only on parse-server cloud code starting 2.6.2*
+
+Sometimes you may want monitor Live Query Event to be used with a 3rd Party such as datadog. The `onLiveQueryEvent` trigger can be logging events triggered, number of clients connected, number of subscriptions and errors.
```
Comment (TomWFox) [inline _includes/cloudcode/cloud-code.md]:
```suggestion
Sometimes you may want to monitor Live Query Events to be used with a 3rd Party such as datadog. The `onLiveQueryEvent` trigger can log events triggered, number of clients connected, number of subscriptions and errors.
```

**Review by TomWFox — COMMENTED** [review]
Nice!

**Comment by dplewis** [comment]
@davimacedo Are there any other features you can think of that we can add to this for better monitoring?

**Comment by davimacedo** [comment]
I think it would be very helpful to have the client's `installationId`, `sessionToken`, and `usingMasterKey` in these events. I've seen some developers asking for them.

**Comment by dplewis** [comment]
I opened a PR for improving this. https://github.com/parse-community/parse-server/pull/5927

I will update this PR and the Wiki after the next server release.

**Comment by dplewis** [comment]
@TomWFox i forgot about this one. I already updated the Wiki. This should be good to go after the last release.

File: _includes/cloudcode/cloud-code.md [hunk 2]
```diff
@@ -469,6 +469,44 @@ Parse.Cloud.beforeLogin(async request => {
...
+  subscriptions,
+  error
+}) => {
+  if (event !== 'ws_disconnect') {
+    return;
+  }
+  // Do your magic
+});
+```
+*client, sessionToken, useMasterKey and installationId are available on parse-server cloud code 2.8.0+*
```
Comment (TomWFox) [inline _includes/cloudcode/cloud-code.md]: 3.8.0?
```suggestion
*client, sessionToken, useMasterKey and installationId are available on parse-server cloud code 3.8.0+*
```

**Review by TomWFox — CHANGES_REQUESTED** [review]

**Pull request merged**